src/components/LLayerGroup.vue

```diff
@@ -0,0 +1,37 @@
+<script>
+import { onMounted, ref, inject } from "vue";
+import { remapEvents, propsBinder } from "../utils.js";
+import { props, setup as layerGroup } from "../functions/layerGroup";
```

Comment:
```suggestion
import { props, setup as layerGroupSetup } from "../functions/layerGroup";
```
I prefer a longer name to having two different variables named `layerGroup` in the file, depending whether you're in the `onMounted` function or not. Also more consistent with other components to use the `Setup` suffix. WDYT?